client/state/ui/test/actions.js

```diff
@@ -7,38 +7,125 @@ import { expect } from 'chai';
  * Internal dependencies
  */
 import {
-	SELECTED_SITE_SET,
+	NOTIFICATIONS_PANEL_TOGGLE,
+	PREVIEW_IS_SHOWING,
 	ROUTE_SET,
-	NOTIFICATIONS_PANEL_TOGGLE
+	SECTION_SET,
+	SELECTED_SITE_SET
 } from 'state/action-types';
-import { setSelectedSiteId, setRoute, toggleNotificationsPanel } from '../actions';
+import {
+	setAllSitesSelected,
+	setPreviewShowing,
+	setRoute,
+	setSection,
+	setSelectedSiteId,
+	toggleNotificationsPanel
+} from '../actions';
 
 describe( 'actions', () => {
-	describe( '#setSelectedSiteId()', () => {
-		it( 'should return an action object', () => {
-			const action = setSelectedSiteId( 2916284 );
+	describe( 'setAllSitesSelected()', () => {
+		it( 'should return an action object with a null siteId', () => {
+			const action = setAllSitesSelected();
 
 			expect( action ).to.eql( {
 				type: SELECTED_SITE_SET,
-				siteId: 2916284
+				siteId: null
+			} );
+		} );
+	} );
+
+	describe( 'setPreviewShowing()', () => {
+		it( 'should return an action object where isShowing equals to true', () => {
+			const action = setPreviewShowing( true );
+
+			expect( action ).to.eql( {
+				type: PREVIEW_IS_SHOWING,
+				isShowing: true
+			} );
+		} );
+
+		it( 'should return an action object where isShowing equals to false', () => {
+			const action = setPreviewShowing( false );
+
+			expect( action ).to.eql( {
+				type: PREVIEW_IS_SHOWING,
+				isShowing: false
 			} );
 		} );
 	} );
 
 	describe( 'setRoute()', () => {
-		it( 'should return an action object', () => {
-			const action = setRoute( '/foo' );
+		const route = '/foo';
+
+		it( 'should return an action object with path and no query arguments', () => {
```

Comment:
Maybe `should return and action with an empty query object if no query is supplied`?

Reply:
:+1: c8ab6f317f174ec44e50af301a0f36f53a2d4cad